test/test_primitives/assert_helper.h

```diff
@@ -29,6 +29,37 @@ inline void assert_bounding_box(const Collection<T> &collection, const unsigned
   CHECK(bmax[2] == Approx(true_max[2]));
 }
 
+/**
+    If multiple IDs (more objects) are used, all are checked and a global minimum and maximum are searched for
+    @param num_prim_id : Number of prim_ids in object
+*/
+template<typename T, template<typename> typename Collection>
+inline void assert_bounding_box_multi_prim_id(const Collection<T> &collection, const unsigned int prim_id,
+                                              const Vec3r<T> &true_min, const Vec3r<T> &true_max,
+                                              const unsigned int num_prim_id) {
+
+  const auto [bmin_, bmax_] = collection.get_primitive_bounding_box(prim_id);
+
+  Vec3r<T> bmin{bmin_[0], bmin_[1], bmin_[2]};
+  Vec3r<T> bmax{bmax_[0], bmax_[1], bmax_[2]};
```

Comment:
initialize  bmax and bmin with sensible values (imho: bmax -> -inf, bmin = inf) instead of calling the bounding box functions this way.

Reply:
I dont unterstand where the inizilization is bmax = -inf ?

Reply:
bmin_ and bmax_ are only used to find a first set of values for and bmax = -inf guarantees that every other value is larger, therefore setting the maximum in the first step.

Reply:
okay, should be fixed now